sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -678,44 +678,92 @@ public async Task CanQueryWithFormattedQuery(FormattableStringWrapper query)
 
         [LiveOnly]
```

Comment:
Does this still need to be LiveOnly? I think it was LiveOnly because we were querying the resource group resourceId which had a lot of data, I would expect the Workspace resource Id to not have much data.